app/assets/stylesheets/scaffolds.scss

```
@@ -0,0 +1,65 @@
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px; }
+
```

```suggestion
body {
  background-color: #fff;
  color: #333;
  margin: 33px;
}
```
閉じタグはこのようにするのが一般的になるので、以降の記述で意識するとgoodです:)